pkg/api/redis/v1/const.go

```diff
@@ -2,5 +2,7 @@ package v1
 
 const (
 	// ClusterNameLabelKey Label key for the ClusterName
-	ClusterNameLabelKey string = "redis-operator.io/cluster-name"
+	ClusterNameLabelKey string = "redis-operator.k8s.io/cluster-name"
+	// ClusterPodSpecLabelKey label key for the PodSpec MD5 hash
+	ClusterPodSpecLabelKey string = "redis-operator.k8s.io/podspec-md5"
```

Comment:
name with MD5 inside (and not Cluster)? 
PodSpecMD5LabelKey

Reply:
OK